frontend/src/app/components/NewsletterFooter/index.scss

```diff
@@ -4,41 +4,44 @@
 $email-icon--width: 74px;
 $email-icon--height: 80px;
 $email-icon--responsive-unit: .66;
+$error-border: #c4c4c4;
+$error-background: #f2f2f2;
 
 
 .newsletter-footer {
   background: $transparent-white-1;
   padding: ($grid-unit * 1.75) 0;
 
   .error {
-    color: $red-70;
-    border: 1px solid #C4C4C4;
+    background-color: $error-background;
+    border: 1px solid $error-border;
     border-radius: 4px;
-    background-color: #F2F2F2;
+    color: $red-70;
     font-size: $font-unit * 7 / 6;
     line-height: 8 / 7;
     margin-bottom: $grid-unit;
     padding: $grid-unit * .5;
   }
 
   header {
-    width: 45%;
     @include respond-to ('big') {
       margin-left: 55px;
     }
+
+    width: 45%;
   }
 
   form {
-    width: 45%;
-    max-width: 360px;
     @include respond-to ('big') {
       margin-right: 55px;
     }
+
+    max-width: 360px;
+    width: 45%;
   }
 
   header {
```

Comment:
Should this be combined w/ the `header {}` block up on L26-32?

Reply:
I'll fix this in #3114